src/keboola_mcp_server/tools/components/model.py

```diff
@@ -289,6 +290,7 @@ class ComponentConfigurationOutput(BaseModel):
         description='The component this configuration belongs to',
         default=None,
     )
+    links: list[Link] = Field(..., description='The links relevant to the tool call.')
```

Comment:
this description is copy-pasted everywhere. IMHO it should be specialized for each object, in this case something like `The links relevant to the configuration`.